test/api/test_workflows.py

```diff
@@ -2200,6 +2200,7 @@ def test_optional_workflow_output(self):
     state:
       produce_out_1: False
       filter_text_1: '1'
+      produce_collection: True
```

Comment:
I think you need to set this to False here. This is the default, but if `state` is provided in gxformat2 workflows missing parameters aren't updated on 18.09.